docs/servers/add-gcp-server.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Add a Server to Google Cloud Platform (GCP)
+intro: Devopness allows you to provision a server on Google Cloud Platform using
+Google Compute Engine and manage it through Devopness web application.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+# Add a Server to Google Cloud Platform (GCP)
+1. On Devopness, navigate to a project then select an environment
+2. If there's no other server in the selected environment, just click Add Server
+3. If a list of resource cards is displayed, find the Servers card and click View on it
+4. On the upper-right corner of the list click `ADD SERVER`
+5. Choose `Google Cloud Platform` as cloud provider and select its credentials
+6. Provide the instance type information and click `NEXT`
+7. Select your server name, OS and hostname and click `NEXT`
```

Comment:
@mlpbraga I'd suggest:
1. add instructions to select a `region`
1. instead of `Provide the instance` instruct the reader to select an instance from the drop down menu
1. Instead of `Select your server name` use `provide`, cause there aren't any server names to be selected